nusight2/src/client/components/localisation/robot_model.ts

```diff
@@ -168,18 +168,17 @@ export class LocalisationRobotModel {
   }
 
   /* Field to torso rotation in field space. */
```

Comment:
Which I guess would be:

```suggestion
  /* Torso to field rotation in torso space. */
```

Been a while since I've dealt with these... but is it typical for translation and rotation to be in separate spaces? 🤨

Reply:
They are in the same space. 

![image](https://github.com/NUbots/NUbots/assets/41043317/956ab276-b6a5-49cd-a574-40cc0817223f)

Reply:
So comment is correct as is

Reply:
<img width="649" alt="Screen Shot 2023-05-21 at 12 35 11 pm" src="https://github.com/NUbots/NUbots/assets/132620/e1599c50-5f02-4bc2-8a35-0de3e11f92ad">

Since its named `Rft`, doesn't that mean we're going from torso to field?

Reply:
The new code `Rft` is absolutely converting vectors in torso space to field space
so it is "Torso to field"

Reply:
I guess it depends on what context when considering the statement X to Y. Imo, since we are using these functions to position and orientation the robot in field space, it makes sense to leave the comment as is, and not interpret the comment as the transform but as the orientation information it provides.

Reply:
But the information it provides is the transform 🤨

Reply:
`Rft` describes a rotation from torso space to field space. Describing it as anything else is just confusing.

Reply:
If you did want to flip the language you could say that Rft describes the torso basis in field space

Reply:
I agree but the very reason this PR exists and some others in the past is because the language "X to Y rotation" is very easily misinterpreted.

Reply:
I will update comment